### PR TITLE
Fix coverity warning in test-string-util.c

### PR DESCRIPTION
**Pull request by bluca**
CID 1446368:  Incorrect expression  (MISSING_COMMA)

### CODE REVIEW

File: src/test/test-string-util.c [hunk 1]
```diff
@@ -917,7 +917,7 @@ static void test_strverscmp_improved(void) {
                 "123-3.1",
                 "123^patch1",
                 "123^1",
-                "123.a-1"
+                "123.a-1",
```
Comment (bluca) [inline src/test/test-string-util.c:920]: @yuwata I assume this was unintentional?

Otherwise we can wrap it in brackets to make coverity happy.

**Comment by yuwata** [comment]
Thanks. LGTM. But the same fix is already included in @keszybz's PR https://github.com/systemd/systemd/pull/11484...

**Comment by bluca** [comment]
> Thanks. LGTM. But the same fix is already included in @keszybz's PR #11484...

Ah, missed that. If that one's review concludes today, I'll close this.

**Comment by keszybz** [comment]
Let's merge this. One warning less.

**Pull request merged**